tests/dataset/test_batch.py

```diff
@@ -20,6 +20,18 @@ def test_model(configuration):  # noqa: D103
     _ = Batch.fake(configuration=configuration)
 
 
+def test_model_nwp_channels(configuration):  # noqa: D103
+
+    configuration.input_data = configuration.input_data.set_all_to_defaults()
+    configuration.process.batch_size = 4
+    configuration.input_data.nwp.nwp_channels = ["dlwrf"]
```

Comment:
Tiny typo in the code (here and below): `dlwrf` should be `dswrf` (short for downwards short-wave radiation flux`, I think!)